app/src/main/res/values/colors.xml

```diff
@@ -116,4 +116,9 @@
     <item>#FF0097</item>
     <item>#1E7145</item>
   </array>
+  
+  <!-- Account Manager -->
```

Comment:
Is it really necessary to add dedicated colors for this screen? Aren't there any already used colors that could be reused?

For example I can see that there is already `appshortcut_swipe_delete` which uses the same red color. You could rename it to something like `destructive_action_background` and use that instead. Generally I would suggest adding as few new colors as possible to make it easier to implement light theme in future.